server/src/routes/follow.js

```diff
@@ -0,0 +1,42 @@
+const express = require("express");
+const Follows = require('../db/mongo/models/follow');
+
+const router = express.Router();
+
+router.get('/:email_from/followers', async (req, res) => {
+    const emailFrom  = req.params.email_from;
+    try {
+        let followers = await Follows.find({ email_from: emailFrom}).exec();
```

Comment:
Might consider moving it to `follow.js` in services package for standardization (even though it's short), same for all other mongoose queries